cmake/linker/ld/target.cmake

```diff
@@ -2,5 +2,57 @@
 
 find_program(CMAKE_LINKER     ${CROSS_COMPILE}ld      PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
```

Comment:
@ozhuraki I am fine with that, but I think it should be a separate PR:
1. From isolating commits across CI testing.
2. It seems one would want to apply that change not only to the linker, but also compiler and binutils.

I guess you want that in relation to posix-based arches?

Reply:
@mped-oticon 

> I am fine with that, but I think it should be a separate PR.

Thanks and OK.

> It seems one would want to apply that change not only to the linker, but also compiler and binutils.
> I guess you want that in relation to posix-based arches?

For native_posix clang and for host-gcc it’s already like that.

It would be nice to have more flexibility for the linker too.